### ISSUE TITLE
Add disk space display to admin page

### ISSUE THREAD

**Issue by spatiald** [rachelproject/contentshell#22]
Helpful when user wants to upload content and they don't know how much room they have left.  It would be useful to display a "Disk Space" tab before? the "Stats" tab that shows a graphical bar for each disk (like Windows Disk Management).  Something like this (with colors and Jonathan's style):

*This is CAP example but something similar could be done for the Pi*

### Admin Partition (preloaded)
- content zip files upload location
```
|||||||||||||||||||||||||
```
### Teacher Partition (uploaded)
- individual files upload location
```
|||||||
```
### RACHEL Partition (RACHEL modules)
- RACHEL module location
```
||||||||||||||||||||||||||||||||||||||||||||||||||
```

**Comment by needlestack** [comment]
This is a great idea - and fits right in to what I'm doing at the moment :)

**Comment by needlestack** [comment]
Included in v2.0.0
